xdsl/dialects/pdl.py

```diff
@@ -853,7 +788,7 @@ class RewriteOp(IRDLOperation):
     """
 
     name = "pdl.rewrite"
-    root = opt_operand_def(OperationType)
+    root = opt_operand_def(OperationType())
```

Comment:
I thought we changed this to not matter?

Reply:
Removed the change, thanks I forgot about it